src/components/schema/_schema_item.scss

```diff
@@ -0,0 +1,79 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+.ouiSchemaItem {
+  display: flex;
+  // sass-lint:disable-block no-misspelled-properties
+  gap: $ouiSizeS;
+  align-items: center;
+  padding: $ouiSizeS;
+  border-radius: $ouiBorderRadius;
+
+  &:hover {
+    @include ouiFocusBackground($ouiTextColor);
+
+    .ouiSchemaItem__actions .ouiButtonIcon {
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+}
+
+.ouiSchemaItem__icon {
+  align-self: flex-start;
+  margin: $ouiSize / 2 0; // To align with buttonIcon
+}
+
+.ouiSchemaItem__label {
+  flex: 1;
+  margin: $ouiSize / 2 0; // To align with buttonIcon
+}
+
+.ouiSchemaItem__actions {
+  // sass-lint:disable-block no-misspelled-properties
+  gap: $ouiSizeXS;
+  display: flex;
+
+  .ouiButtonIcon {
+    opacity: 0;
+    transition: opacity $ouiAnimSpeedNormal $ouiAnimSlightResistance;
+
+    &:focus {
+      opacity: 1;
+    }
+  }
```

Comment:
Is there not an existing mixin to use for this?

Reply:
I copied this over from ListGroup. I dont think it used one, but if there is one I will update it